src/main.cpp

```diff
@@ -1,5 +1,8 @@
 #include <pybind11/pybind11.h>
 
+#define STRINGIFY(x) #x
+#define MACRO_STRINGIFY(x) STRINGIFY(x)
```

Comment:
I haven't come across this before ... why is the 2nd macro needed? Is this working around compiler bugs?

Reply:
This is an idiom if you need to stringify the contents post macro expansion. Just the first one would not expand the macro inside. It's `BOOST_PP_STRINGIZE`, basically.